test/integrations/destinations/facebook_pixel/processor/configLevelFeaturesTestData.ts

```diff
@@ -0,0 +1,483 @@
+import { VERSION } from '../../../../../src/v0/destinations/facebook_pixel/config';
+import { overrideDestination, generateTrackPayload } from '../../../testUtils';
+
+const commonDestination = {
```

Comment:
Use the `Destination` Type for this variable

Reply:
added